package.json

```
@@ -16,6 +16,9 @@
     "build": "tsc -p tsconfig.json",
     "build:watch": "tsc -p tsconfig.json --watch",
     "dev": "nodemon --exec \"npm start\"",
+    "docker:build": "docker build -t probot-auto-merge-alpine .",
```

This doesn't align with what is defined in the readme.

```suggestion
    "docker:build": "docker build -t probot-auto-merge .",
```

---

Oh, thanks for the catch!